locales/en/plugin__kubevirt-plugin.json

```diff
@@ -1058,16 +1065,23 @@
   "Status conditions": "Status conditions",
   "Stop": "Stop",
   "Storage": "Storage",
+  "Storage checkup": "Storage checkup",
+  "Storage checkup details": "Storage checkup details",
+  "Storage checkup validating storage is working correctly for VirtualMachines using the kiagnose engine.": "Storage checkup validating storage is working correctly for VirtualMachines using the kiagnose engine.",
   "Storage class": "Storage class",
   "Storage Class": "Storage Class",
+  "Storage class with empty claimPropertySets": "Storage class with empty claimPropertySets",
   "Storage IOPS": "Storage IOPS",
   "Storage IOPS total read / write": "Storage IOPS total read / write",
+  "Storage missing VolumeSnapShootClass": "Storage missing VolumeSnapShootClass",
```

Comment:
I think the extra 'o' in 'Shoot' is a typo.

Reply:
I think is `VolumeSnapshotClass`

Reply:
done